jenkins-slave.yml

```diff
@@ -45,6 +45,9 @@ plugins:
     username: jenkins
     groups:
       - docker
+  pip_install:
+    packages:
+      - docker-compose==1.7.1
   ec2_publish:
-    manifest_url: https://pebble-amis.s3-us-west-2.amazonaws.com/jenkins.json
+    manifest_url: https://ami.pbl.io.s3-us-west-2.amazonaws.com/jenkins/latest.json
     public: true
```

Comment:
I set it up with S3+CNAME yesterday; maybe CloudFront+ACM for SSL? (is that a thing?)